contracts/MarginCalculator.sol

```diff
@@ -32,26 +32,33 @@ contract MarginCalculator {
     }
 
     /**
-     * @notice Return the cash value of an expired oToken.
-     * @dev For call return = Max (0, ETH Price - oToken.strike)
-     * @dev For put return Max(0, oToken.strike - ETH Price)
+     * @notice Return the net worth of an expired oToken in collateral.
      * @param _otoken otoken address
-     * @return the cash value of an expired otoken
+     * @return the exchange rate that shows how much collateral unit can be take out by 1 otoken unit, scaled by 1e18
      */
-    function getExpiredCashValue(address _otoken) public view returns (uint256) {
-        require(_otoken != address(0), "MarginCalculator: Invalid token address.");
+    function getExpiredPayoutRate(address _otoken) external view returns (uint256) {
+        uint256 cashValueInStrike = _getExpiredCashValue(_otoken);
+
         OtokenInterface otoken = OtokenInterface(_otoken);
```

Comment:
We can move this casting in the first line of this function, and pass `OtokenInterface` as argument also to `_getExpiredCashValue()` as it cast the address there also.